packages/zambdas/ottehr-spec.json

```diff
@@ -1061,7 +1061,13 @@
     "OTTEHR_ORGANIZATION": {
       "resourceType": "Organization",
       "active": true,
-      "name": "Example Organization"
+      "name": "Example Organization",
+      "telecom": [
+        {
+          "system": "phone",
+          "value": "#{var/fax-number}"
```

Comment:
if this is a new var, please also add it to `packages/zambdas/.env/local.template.json`. but also, should fax number acquisition be IaC driven? 😅

Reply:
a5afdf1

Reply:
terraform generally runs on a "create read update delete" policy. update is optional if there are no updatable properties or updating those properties requires deleting and recreating the resource on the provider's side. the only other requirement is that we can construct a consistent return value for "state".

i'm aware of the onboard and offboard endpoints, is there one to just get info about the fax number you have?

Reply:
while that's _technically_ enough for me to make a terraform provider, i'd rather wait until i have an endpoint that gives me a 200 OK :)